src/modules/users/infrastructure/repositories/__tests__/user.repository.soft-delete.spec.ts

```
@@ -0,0 +1,287 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { DataSource } from 'typeorm';
+import { UserTypeormRepository } from '../user.typeorm.repository';
+import { User } from '../../../domain/entities/user.entity';
```

ESLint

[error] 'User' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)